Resources/doc/DSL/ManageLocation.yml

```diff
@@ -58,6 +58,7 @@
     is_hidden: true|false # Optional, Set the visibility of the location
     sort_field: x # Optional
     sort_order: ASC|DESC # Optional
+    remote_id: # Optional, Set the remote_id of the location
```

Comment:
There is only one thing I am not 100% happy about this: it kind of clashes with the DSL for contents where we use remote_id as matching key...